source/applications/advanced/color_balance.py

```diff
@@ -26,6 +26,19 @@ class MeanColor:
     blue: np.array
 
 
+def _frame2d_to_rgb(frame: zivid.Frame2D):
```

Comment:
Technically "_frame2d_to_rgba"

Reply:
If I accept this comment then I think I have to replace all 'rgb' with 'rgba' in this sample and other samples. What do you think?

Reply:
I think that if an array is "HxWx4" RGBA data, then it should be called "rgba". Calling it "rgb" could cause confusion.

Reply:
done